components/ui/sosial-media/render-post-card.tsx

```
@@ -111,9 +111,11 @@
           {query ? (
             <View className="ml-3">
               <Text className="text-[16px] font-semibold text-text-primary">
-                {item.user.profile.full_name.length > 170
-                  ? `${item.user.profile.full_name.slice(0, 170)}...`
-                  : item.user.profile.full_name}
+                {item.user.profile && item.user.profile.full_name
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)